.github/CHANGELOG.md

```diff
@@ -2,6 +2,50 @@
 
 <h3>New features since last release</h3>
 
+* Custom gradient transforms can now be created using the new
+  `@qml.gradients.gradient_transform` decorator on a batch-tape transform.
+  [(#1589)](https://github.com/PennyLaneAI/pennylane/pull/1589)
+
+  Quantum gradient transforms are a specific case of `qml.batch_transform`.
+  All quantum gradient transforms accept a tape, and output
```

Comment:
It looks like quantum gradient transforms also accept QNodes in the example below (line `38`)?

Reply:
good point! I've edited this accordingly.

Reply:
Ah, I think what I was trying to say here is 'To _create_ a custom gradient transform, simply write a function that accepts a tape...'

Reply:
have edited this to make it more clear :)